torch/_torch_docs.py

```diff
@@ -44,15 +51,16 @@ def merge_dicts(*dicts):
 """})
 
 single_dim_common = merge_dicts(reduceops_common_args, parse_kwargs("""
-    dim (int): the dimension to reduce
+    dim (int): the dimension to reduce.
 """), {'keepdim_details': """If :attr:`keepdim` is ``True``, the output tensor is of the same size
 as :attr:`input` except in the dimension :attr:`dim` where it is of size 1.
 Otherwise, :attr:`dim` is squeezed (see :func:`torch.squeeze`), resulting in
 the output tensor having 1 fewer dimension than :attr:`input`."""})
 
 
 factory_common_args = parse_kwargs("""
-    out (Tensor, optional): the output tensor
+    input (Tensor): the input tensor.
+    out (Tensor, optional): the output tensor.
```

Comment:
If these are the same as `common_args`, I don't think you need to repeat this. You can call [update()](https://docs.python.org/3/library/stdtypes.html#dict.update) to incorporate `common_args` at the end of this statement.

Reply:
I think this applies to `reduceops_common_args` too